src/beyondllm/llms/together_ai.py

```diff
@@ -0,0 +1,66 @@
+from beyondllm.llms.base import BaseLLMModel, ModelConfig
+from typing import Any, Dict
+from dataclasses import dataclass, field
+import os
+from together import Together
+
+@dataclass
+class TogetherModel:
+    """
+    Class representing a Language Model (LLM) using Together AI.
+
+    Example:
+    ```
+    >>> llm = TogetherModel(api_key="<your_api_key>", model_kwargs={"temperature": 0.7})
+    ```
+    or
+    ```
+    >>> import os
+    >>> os.environ['TOGETHER_API_KEY'] = "***********" #replace with your key
+    >>> llm = TogetherModel()
+    ```
+    """
+    api_key: str = " "
+    model_name: str = "meta-llama/Llama-3-8b-chat-hf"
+    model_kwargs: dict = field(default_factory=lambda: {
+        "temperature": 0.7,
+        "top_p": 0.9,
+        "max_tokens": 1024,
+    })
+
+    def __post_init__(self):
+        if not self.api_key:
+            self.api_key = os.getenv('TOGETHER_API_KEY')
+            if not self.api_key:
+                raise ValueError("TOGETHER_API_KEY is not provided and not found in environment variables.")
+        self.load_llm()
+
+    def load_llm(self):
+        """Load the Together client."""
+        try:
+            self.client = Together(api_key=self.api_key)
+        except Exception as e:
+            raise Exception(f"Failed to initialize Together client: {str(e)}")
+
+    def predict(self, prompt: Any) -> str:
+        """Generate a response from the model based on the provided prompt."""
+        try:
+            stream = self.client.chat.completions.create(
```

Comment:
by default, its ideal to keep stream as False. please change it as per cohere implementation you have done

Reply:
Done

Reply:
@madhavi-peddireddy 
Congrats on being approved for the Hacktoberfest swags!  Please share your details with me so we can send it over to you.

You can either drop the details in our Discord channel: bit.ly/aiplanet-discord or email me at shivaya@aiplanet.com.